posthog/hogql/database/models.py

```diff
@@ -23,17 +24,34 @@ class Config:
     array: Optional[bool] = None
     nullable: Optional[bool] = None
 
+    def is_nullable(self) -> bool:
+        return not not self.nullable
+
+    def get_constant_type(self) -> "ConstantType":
+        from posthog.hogql.ast import UnknownType
+
+        return UnknownType(nullable=self.is_nullable())
+
 
 class IntegerDatabaseField(DatabaseField):
-    pass
+    def get_constant_type(self) -> "ConstantType":
+        from posthog.hogql.ast import FloatType
+
+        return FloatType(nullable=self.is_nullable())
```

Comment:
Should this not return `IntegerType`?

Reply:
Lol, that will probably improve things 🙈